flycheck.el

```diff
@@ -3610,6 +3650,55 @@ list."
       ;; highlighting
       (flycheck-error-list-highlight-errors preserve-pos))))
 
+(defvar flycheck-error-list-current-filter nil
+  "The minimum level of errors to display in the error list.
+
+If set to an error level, only displays errors whose error level
+is at least as severe as this one in the error list.  If nil,
+display all errors.")
+
+(defun flycheck-error-list-mode-line-filter-indicator ()
+  "Create a string representing the current error list filter."
+  (if flycheck-error-list-current-filter
+      (format " [>= %s]" (symbol-name flycheck-error-list-current-filter))
+    ""))
+
+(defun flycheck-error-list-query-level (prompt)
```

Comment:
Please rename this one to `read-flycheck-error-level`, for consistency with similar functions and with `read-flycheck-checker`.  Also, please move it below the definition of `read-flycheck-checker`, so that all minibuffer input functions are in one section.